mlflow/utils/databricks_utils.py

```diff
@@ -630,6 +630,18 @@ def get_databricks_workspace_info_from_uri(tracking_uri: str) -> Optional[Databr
         return None
 
 
+def check_databricks_secret_scope_access(scope_name):
+    dbutils = _get_dbutils()
+    if dbutils:
+        try:
+            dbutils.secrets.list(scope_name)
+        except Exception as e:
+            _logger.warning(
+                f"Unable to access Databricks secret scope '{scope_name}'. Please verify that the current"
+                f" Databricks user has 'READ' permission for this scope. Error: {str(e)}"
```

Comment:
Can we clarify that, in order to deploy the model to Databricks Model Serving, relevant keys will need to be set in the scope with name `{scope_name}`? Can we also list the relevant keys and provide a link to the Databricks docs?

Without this, users might have a hard time figuring out what action to take in response to this warning :)